tests/test_grapple.py

```diff
@@ -104,10 +104,10 @@ def test_pages(self):
         self.assertEqual(type(executed["data"]["pages"][0]), dict)
 
         pages_data = executed["data"]["pages"]
-        self.assertEqual(pages_data[0]["contentType"], "home.HomePage")
-        self.assertEqual(pages_data[0]["pageType"], "HomePage")
-        self.assertEqual(pages_data[1]["contentType"], "home.BlogPage")
-        self.assertEqual(pages_data[1]["pageType"], "BlogPage")
+        self.assertEquals(pages_data[0]["contentType"], "testapp.HomePage")
```

Comment:
as far as I recall `assertEquals` was deprecated in favour of `assertEqual` 
edit: found it https://docs.python.org/3/library/unittest.html#deprecated-aliases

Reply:
good catch, updated.